lib/spack/spack/cmd/style.py

```diff
@@ -343,10 +344,15 @@ def prefix_relative(path):
                 run_function, required = tools[tool_name]
                 print_tool_header(tool_name)
 
-                cmd = which(tool_name, required=required)
-                if not cmd:
-                    color.cprint("  @y{%s not in PATH, skipped}" % tool_name)
-                    continue
+                # Bootstrap tools so we don't need to require install
+                with spack.bootstrap.ensure_bootstrap_configuration():
+                    spec = spack.spec.Spec("py-%s" % tool_name)
```

Comment:
These should have version requirements associated with them.  In particular we need `py-isort@4.3.5:` and `mypy@0.900:` (those are the versions that support config in `pyproject.toml`).